test/private-api/alias.e2e-spec.ts

```diff
@@ -83,7 +83,7 @@ describe('Alias', () => {
           .post(`/api/private/alias`)
           .set('Content-Type', 'application/json')
           .send(newAliasDto)
-          .expect(400)
+          .expect(409)
```

Comment:
Hmmm, actually, `Bad Request` seems to be a better error message for a forbidden note ID than `Conflict`. The best error would probably be `Forbidden` in this case.

Reply:
The question is: Do we want to let the end-user know which alias is forbidden?
In this case, I'd say okay 403 Forbidden is fitting.

But if we want no distinguishing between forbidden aliases and already taken aliases, it makes sense to map both to 409 Conflict.